k8s/celery-beat.yaml.j2

```diff
@@ -0,0 +1,40 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ CELERY_SCHEDULER_NAME }}
+spec:
+  replicas: {{ CELERY_SCHEDULER_REPLICAS }}
```

Comment:
The Makefile has `CELERY_SCHEDULER_REPLICA` but here it has an `S` on the end. Might need a fix?

Reply:
Fixed in a4c7d02f6ec2119197cd5267ddd3f23f91264cf3